Ska/arc5gl.py

```diff
@@ -7,6 +7,9 @@
 import pexpect
 
 # Should put in a watchdog timer to exit from arc5gl after a period of inactivity
+import ska_helpers
+
+__version__ = ska_helpers.get_version('Ska.arc5gl')
```

Comment:
Though perhaps remind me why Ska.tdb gets to use __package__ and Ska.arc5gl does not?

Reply:
Ska.tdb is defined in Ska.tdb/Ska/tdb/__init__.py, while Ska.arc5gl is defined in Ska.arc5gl/Ska/arc5gl.py.

In [line 71 of ska_helpers.version](https://github.com/sot/ska_helpers/blob/master/ska_helpers/version.py#L71), it checks how many directories up in the hierarchy it should go to find the git root. In tdb it has to go up one more level.

I think this is not needed anymore, because of the [the following line](https://github.com/sot/ska_helpers/blob/master/ska_helpers/version.py#L71)

Reply:
Thanks!  So up to you if you want to update this PR or leave it.  Also not sure if you want any extra text in the get_version docstring about this or if the comments (which are great!) are enough.